tracer/test/Datadog.Trace.Tools.dd_dotnet.IntegrationTests/Checks/ProcessBasicChecksTests.cs

```diff
@@ -412,18 +414,57 @@ public void ProfilerNotFoundRegistry()
     public void WrongProfilerRegistry()
     {
         SkipOn.Platform(SkipOn.PlatformValue.MacOs);
+        SkipOn.PlatformAndArchitecture(SkipOn.PlatformValue.Linux, SkipOn.ArchitectureValue.ARM64);
         var registryService = MockRegistryService(Array.Empty<string>(), "wrongProfiler.dll");
 
         using var console = ConsoleHelper.Redirect();
 
-        var result = ProcessBasicCheck.CheckRegistry(registryService);
+        var result = ProcessBasicCheck.CheckRegistry("2.14", registryService);
 
         result.Should().BeFalse();
 
         console.Output.Should().NotContain(MissingRegistryKey(ClsidKey));
         console.Output.Should().Contain(Resources.WrongProfilerRegistry(ClsidKey, "wrongProfiler.dll"));
     }
 
+    [SkippableFact]
+    public void LinuxInstallationDirectory()
+    {
+        SkipOn.Platform(SkipOn.PlatformValue.MacOs);
```

Comment:
Why? 🤔 I'm guessing "because the original did"?

Reply:
Well yes, but the tool is not available on MacOs anyway.